src/Widgets/TaskRow.vala

```diff
@@ -38,6 +38,10 @@ public class Tasks.TaskRow : Gtk.ListBoxRow {
         label.wrap = true;
         label.xalign = 0;
 
+        if (component.get_status () == ICal.PropertyStatus.COMPLETED) {
```

Comment:
We already check for completion at the beginning of the `construct` block :)

```suggestion
        if (completed) {
```

Reply:
Good catch! ;)